### PR TITLE
rework README

### PR DESCRIPTION
**Pull request by mancoast**
I want some review before merging these changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,63 +6,63 @@
...
-## What this is
+## Universal Idea
 
-Mandelbulber generates 3-dimensional fractals.
+In summary, Mandelbulber generates 3-dimensional fractals.
 
-Explore trigonometric, hypercomplex, mandelbox, IFS and many other 3D fractals.
+Explore trigonometric, hyper-complex, Mandelbox, IFS, and many other 3D fractals.
 
-Render with a great palette of customized shaders and create stunning images and videos.
```
Comment (mancoast) [inline README.md]: English noun "shaders" raises ambiguity.
We should use "shade" as a verb.
http://www.ling.upenn.edu/~rnoyer/courses/103/Ling10309Pragmatics.pdf

**Comment by mancoast** [comment]
I rebuilt the readme in MS word and exported to markdown using pandoc.
This enables grammar, style, and spell check.
Style check flags the extra caps with the squiggles as header titles.
Should we revert the header titles?

File: README.md (hunk 1 above)
Comment (buddhi1980) [inline README.md]: Definition of shader: https://pl.wikipedia.org/wiki/Shader

Comment (mancoast) [inline README.md]: I like no squiggles in my word documents.
lols, whats the plan here?

Comment (mclarekin) [inline README.md]: My two cents. Drop the shade/shaders.  
Render with a great palette of customizable materials to  create stunning images and videos.

Comment (mancoast) [inline README.md]: the language has been updated, are we good to merge?

**Pull request merged**